templates/actors/character-sheet.hbs

```diff
@@ -36,7 +36,7 @@
                 </li>
                 <li>
                     {{#if labels.background}}
-                        <span title="{{localize 'DND5E.Background'}}">{{labels.background}}</span>
+                        <span data-tooltip="{{localize 'DND5E.Background'}}">{{labels.background}}</span>
```

Comment:
I don't think `localize` is necessary here, the tooltip should translate the strings automatically with just `data-tooltip="DND5E.Background"`.

Reply:
Good point, though I'll still have to leave `localize` in places where we pass in parameters.